src/event_loop.rs

```diff
@@ -88,7 +88,10 @@ pub enum ControlFlow {
     /// Send a `LoopDestroyed` event and stop the event loop. This variant is *sticky* - once set,
     /// `control_flow` cannot be changed from `Exit`, and any future attempts to do so will result
     /// in the `control_flow` parameter being reset to `Exit`.
-    Exit,
+    /// The contained number will be used as exit code, if the platform supports that (this
+    /// _excludes_ ios, android and wasm32), it's an [`u8`] to prevent some weird surprises as
+    /// described in [`std::process::exit`].
+    Exit(u8),
```

Comment:
This should be an `i32`. It shouldn't be winit's decision to impose platform limitations. If anything, I would modify this to say that the value will be passed to `std::process:exit` and link to https://doc.rust-lang.org/std/process/fn.exit.html#platform-specific-behavior to describe platform specific behavior.

Reply:
I thought this could avoid nasty surprises with negative values on unix-alikes, since due to two's complement `-47` would end up as `214` (as it did on my machine), but fair enough if that's not winits problem. I'll commit now, but I'm unable to push anything until Tuesday.